<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Metrics</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_generateReport">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Generate Report</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ple_scanResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>511</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Scan Result</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Basic Controls</string>
   </property>
   <widget class="QLineEdit" name="le_fileFilters">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>498</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File Filter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_srcPath">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_browsePath">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Advanced Controls</string>
   </property>
   <widget class="QLineEdit" name="le_dirFilters">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_skipDirEnable">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Skip Directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="tb_report">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>301</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_updateReport">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Update Report</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_scanResult">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_scanFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Scan Files</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clearAllList">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear All List</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label</zorder>
  <zorder>pb_generateReport</zorder>
  <zorder>ple_scanResult</zorder>
  <zorder>label_4</zorder>
  <zorder>tb_report</zorder>
  <zorder>pb_updateReport</zorder>
  <zorder>cbox_scanResult</zorder>
  <zorder>pb_scanFiles</zorder>
  <zorder>pb_clearAllList</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
